src/commands/org/open.ts

```diff
@@ -36,7 +38,7 @@ export class OrgOpenCommand extends SfCommand<OrgOpenOutput> {
     browser: Flags.option({
       char: 'b',
       summary: messages.getMessage('flags.browser.summary'),
-      options: ['chrome', 'edge', 'firefox'] as const, // These are ones supported by "open" package
+      options: ['chrome', 'edge', 'firefox', 'browserPrivate'] as const, // These are ones supported by "open" package
```

Comment:
I understand this follows the `open` API but not sure if we should expose it in the CLI.

what if you add a new `--private` flag that's exclusive with `--browser`?

Reply:
I almost did it that way.  I'll let your vote tip it to that side.